src/main/java/com/github/underscore/U.java

```diff
@@ -467,34 +462,24 @@ public void forEachRight(final Consumer<? super T> func) {
         return filtered;
     }
 
-    public static <E> Set<E> filter(final Set<E> set, final Predicate<E> pred) {
-        final Set<E> filtered = newLinkedHashSet();
-        for (E element : set) {
-            if (pred.test(element)) {
-                filtered.add(element);
-            }
-        }
-        return filtered;
-    }
```

Comment:
This method was removed. How to filter sets?

Reply:
The filter method, which i implemented, can be calles with Set. This is why I think the new one is more universal, because you don't need to implement the function for Set, Map, List, etc., just one method for every type.